packages/components/psammead-most-read/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.0 | [PR#3116](https://github.com/bbc/psammead/pull/3116) Added `maxTwoColumns` prop to conditionally render GEL_GROUP_5_SCREEN_WIDTH_MIN media queries |
```

Comment:
We need to bump it to 1.1.0, it's on 1.0.7 atm :)

Reply:
🤦‍♂